modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2351,12 +2354,10 @@ export class Wallet implements IWallet {
     const successfulTxs: any[] = [];
     const failedTxs = new Array<Error>();
     for (const unsignedBuild of unsignedBuilds) {
-      // fold any of the parameters we used to build this transaction into the unsignedBuild
-      const unsignedBuildWithOptions: PrebuildAndSignTransactionOptions = _.pick(
-        params,
-        this.prebuildWhitelistedParams()
-      );
-      unsignedBuildWithOptions.prebuildTx = unsignedBuild;
+      const unsignedBuildWithOptions: PrebuildAndSignTransactionOptions = {
+        ...params,
+        prebuildTx: unsignedBuild,
+      };
```

Comment:
we should probably filter based on `prebuildWhitelistedParams`

Reply:
Filtering would remove `walletPassphrase` making us unable to sign the transaction. `sendMany` also calls `prebuildAndSignTransaction` with the full `params` with no issue.